ch02.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[ch02_getting_started]]
+== Getting Started
+
+// TODO intro paragraph - no empty headings!
+
+==== Choosing a Lightning Network Wallet
+
+* full nodes (c-lightning, eclair, lnd) + remote controls
+* phone / desktop wallets (SPV clients)
+* custodial services / wallets?
+// Mastering bitcoin also had a section about custodial web wallets. So it might be fair to include them.
+
+==== Quick Start
+
+* We start with a mobile wallet like eclair
+* Verify the authenticity of the software (e.g. App store / Acinq website / github)
+** discuss that a general phising scheme might consist of tricking you to download a similar looking software
+
+[[getting_first_bitcoin]]
+==== Getting Your First Bitcoin on the Lightning Network
+
+* Trade fiat for Bitcoin (as in Mastering Bitcoin)
+
+[[using_own_bitcoin]]
+==== Process for people who already own Bitcoin ====
+
+* send bitcoin to lightning wallet (1 onchain transaction - soon nodes / wallets may support funding a channel directly without sending bitcoins to the lightning network wallet first)
+* find a node to open a channel with (Node explorer / Autopilots / ...)
+* open a connection
```

Comment:
needs to be done in the onramp case too (even if the other side opens the channel one of the two has to open a connection first)

I remmeber that I found this extremely confusing why I needed to open a connection first when I didn't understand the details of the protocol